uniffi_bindgen/src/bindings/kotlin/gen_kotlin.rs

```diff
@@ -280,4 +280,20 @@ mod filters {
             _ => format!("{}.read({})", type_kt(type_)?, nm),
         })
     }
+
+    /// Generate a Kotlin exception name
+    ///
+    /// This replaces "Error" at the end of the name with "Exception".  Rust code typically uses
+    /// "Error" for any type of error but in the Java word, "Error" means a non-recoverable error
+    /// and is distinguished from an "Exception".
+    pub fn exception_name_kt(error: &Error) -> Result<String, askama::Error> {
+        match error.name().strip_suffix("Error") {
+            None => Ok(error.name().to_owned()),
+            Some(trimmed) => {
+                let mut rv = trimmed.to_owned();
+                rv.push_str("Exception");
+                Ok(rv)
```

Comment:
Purely as a style-consistency nit, I'm not 100% sure how to parse "rv" here - is it "return value"? I think I'd lightly prefer a more specific name like `err_name` or similar, but I don't feel strongly about it (and I'm sure there are plenty of things in this codebase that are not consistent with variable names 😅)

Reply:
Yeah, "rv" was supposed to mean return value.  I just pushed a new commit that has something more descriptive.